src/lib/translations/en/application.json

```diff
@@ -1,5 +1,5 @@
 {
-	"title": "Quilt Developer Wiki :P",
+	"title": "Quilt Developer Wiki",
```

Comment:
Hot take : I think the silly is fine to keep as long as we don't have real content for this page \:P

Reply:
imo the dev wiki seems to get a lot of viewership, and it's better to have it looking spiffy

Reply:
Then... could we maybe get a small welcome thingy just to make the page seem not empty ?

Reply:
yes